charts/sonatype-nexus/values.yaml

```diff
@@ -51,6 +51,7 @@ nexus:
   #   - 192.168.1.10/32
   # labels: {}
   securityContext:
```

Comment:
Can you instead have a nexus.securityContextEnabled above this and leave the ability to add fields. I am concerned that other might be using values like runAsUser or runAsGroup. This would break their upgrade as you are locking it  to only the one field of securityContext.

Reply:
@rjkernick Just to make sure I understand:

```
nexus:
  securityContextEnabled: true
  securityContext:
    runAsUser: 1000
    fsGroup: 2000
```

And then in the template revert things back to how they were, with the exception of the if statement looking something like

```
{{- if .Values.nexus.securityContextEnabled }}
```

Reply:
@rjkernick Requested changes have been made. Let me know if this is appropriate.